<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="7" total="9" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Smoke Suite" duration-ms="14131" started-at="2019-06-18T23:02:34Z" finished-at="2019-06-18T23:02:48Z">
    <groups>
      <group name="smoke">
        <method signature="LoginPageTest.loginToOrangeHRM()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="loginToOrangeHRM" class="com.syntax.testcases.LoginPageTest"/>
        <method signature="LoginPageTest.userLogin()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="userLogin" class="com.syntax.testcases.LoginPageTest"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Smoke Test" duration-ms="14131" started-at="2019-06-18T23:02:34Z" finished-at="2019-06-18T23:02:48Z">
      <class name="com.syntax.testcases.AddEmployeeTest">
        <test-method status="PASS" signature="generateReport()[pri:0, instance:com.syntax.testcases.AddEmployeeTest@e25b2fe]" name="generateReport" is-config="true" duration-ms="203" started-at="2019-06-18T23:02:34Z" finished-at="2019-06-18T23:02:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReport -->
        <test-method status="PASS" signature="flushReport()[pri:0, instance:com.syntax.testcases.AddEmployeeTest@e25b2fe]" name="flushReport" is-config="true" duration-ms="329" started-at="2019-06-18T23:02:48Z" finished-at="2019-06-18T23:02:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushReport -->
      </class> <!-- com.syntax.testcases.AddEmployeeTest -->
      <class name="com.syntax.testcases.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="setUp" is-config="true" duration-ms="4446" started-at="2019-06-18T23:02:34Z" finished-at="2019-06-18T23:02:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginToOrangeHRM()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="loginToOrangeHRM" duration-ms="4028" started-at="2019-06-18T23:02:39Z" finished-at="2019-06-18T23:02:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToOrangeHRM -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="tearDown" is-config="true" duration-ms="84" started-at="2019-06-18T23:02:43Z" finished-at="2019-06-18T23:02:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="setUp" is-config="true" duration-ms="1939" started-at="2019-06-18T23:02:43Z" finished-at="2019-06-18T23:02:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="userLogin()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="userLogin" duration-ms="2632" started-at="2019-06-18T23:02:45Z" finished-at="2019-06-18T23:02:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.syntax.testcases.LoginPageTest.userLogin(LoginPageTest.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.syntax.testcases.LoginPageTest@464bee09]" name="tearDown" is-config="true" duration-ms="81" started-at="2019-06-18T23:02:48Z" finished-at="2019-06-18T23:02:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.syntax.testcases.LoginPageTest -->
    </test> <!-- Smoke Test -->
  </suite> <!-- Smoke Suite -->
</testng-results>
